### PR TITLE
calc percent for categoricals

### LINKED ISSUE #3099: Categorical market cards percentage is off by 10x
image

percentage looks to be same as price. in the example above `Wales` has .55% price the value should say 55% and the dot should be located at 55% position.

### PR DESCRIPTION
**Pull request by phoebemirman**
closes https://github.com/AugurProject/augur/issues/3099

### CODE REVIEW

File: packages/augur-ui/src/modules/market-cards/common.tsx [hunk 1]
```diff
@@ -29,17 +29,22 @@ export interface OutcomeProps {
...
+  max: number;
 }
 
-export const Outcome = (props: OutcomeProps) => (
-  <div className={classNames(Styles.Outcome, {[Styles.invalid]: props.invalid, [Styles[`Outcome-${props.index}`]]: !props.invalid})}>
-  	<div>
-    	<span>{props.description}</span>
-    	<span>{props.lastPricePercent.formatted}%</span>
+export const Outcome = (props: OutcomeProps) => {
+  const percent = calculatePosition(props.min, props.max, props.lastPricePercent.value);
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market-cards/common.tsx:38]: If maxPriceBigNumber and minPriceBigNumber is used then the component already has big numbers to work with

**Review by bthaile — APPROVED** [review]
lgtm

File: packages/augur-ui/src/modules/market-cards/common.tsx (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/market-cards/common.tsx:38]: looks good

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🧡
 To view it, be sure to [join Augur Project](https://dashboard.featurepeek.com/login) on FeaturePeek.

 This link will always point to this PR's latest changes: https://peek.run/w9jh4vx

**Pull request merged**